Fabo/0.33.0

faboweb opened the pull request:
Closes #2103 

Updates to SDK 0.33.0
Improves on local testnet generation and usage. Always sends tokens to a fixed address: "cosmos1ek9cd8ewgxg9w5xllq9um0uf4aaxaruvcw4v9e"

Needs the backend to be updated

**Description:**

<!-- Briefly describe what you're adding or fixing with this PR -->

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

fedekunze commented on app/src/renderer/components/transactions/TmLiAnyTransaction.vue:
```diff
@@ -0,0 +1,100 @@
+<template>
```
We renamed this file to LiAnyTransaction. Why is this being changed ?

faboweb commented:
It's not. Merge conflict. I removed the file again.

Federico Kunze <notifications@github.com> schrieb am Sa., 9. März 2019,
19:31:

> *@fedekunze* commented on this pull request.
> ------------------------------
>
> In app/src/renderer/components/transactions/TmLiAnyTransaction.vue
> <https://github.com/cosmos/voyager/pull/2225#discussion_r264009447>:
>
> > @@ -0,0 +1,100 @@
> +<template>
>
> We renamed this file to LiAnyTransaction. Why is this being changed ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cosmos/voyager/pull/2225#pullrequestreview-212571893>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFmO2U3DIPobRYRtOKy4c9NaAhMl1sr4ks5vU_4AgaJpZM4bmvpg>
> .
>

jbibla commented on app/src/renderer/scripts/time.js:
```diff
@@ -2,13 +2,13 @@
-  if (type === `cosmos-sdk/Undelegate`) {
-    const unbondingDelegation = unbondingDelegations[value.validator_addr]
+  if (type === `cosmos-sdk/MsgUndelegate`) {
+    const validatorUnbondingDelegation = unbondingDelegations[value.validator_address]
```
is `validatorUnbondingDelegation` being used?

the naming is a bit confusing too. 

`validatorUnbondingDelegation = unbondingDelegations[value.validator_address]`
`unbondingDelegation = validatorUnbondingDelegation.find()`

jbibla commented on app/src/renderer/vuex/modules/delegation.js:
```diff
@@ -159,11 +156,11 @@ export default ({ node }) => {
       await dispatch(`sendTx`, {
         type: `postDelegation`,
-        to: wallet.address, // TODO strange syntax
+        to: session.address, // TODO strange syntax
```
still strange syntax?

faboweb replied:
yep, just changed the source of the address

fedekunze replied:
what's the strange syntax? can we create an issue ?

faboweb replied:
there is already an issue around refactoring the send module #1828

jbibla commented on package.json:
```diff
@@ -36,7 +36,7 @@
     "postcheckout": "yarn",
     "watch": "tasks/watch.sh",
-    "nodes": "node ./tasks/testnet.js",
+    "nodes": "node ./tasks/local-testnet/start.js",
```
👍

fedekunze commented:
hmm I still get the error from undelegations: ` Error fetching delegations: Cannot read property 'amount' of undefined`

faboweb commented:
@fedekunze could you re-eval please?

fedekunze commented:
> none of the actions work

This is still failing: (re)delegation, unbonding, proposals, send, etc

faboweb commented:
> This is still failing: (re)delegation, unbonding, proposals, send, etc

will fix this

faboweb commented:
Need to check version again. The commit stated 0.32.0. Maybe need to upgrade again.

fedekunze commented:
Karoly was this manually tested ?

fedekunze commented:
@sabau @faboweb this wasn't properly tested and it broke the transactions page
